charts/descheduler/templates/cronjob.yaml

```diff
@@ -39,10 +39,12 @@ spec:
                 - "--policy-config-file"
                 - "/policy-dir/policy.yaml"
                 {{- range $key, $value := .Values.cmdOptions }}
+                  {{- if ne (kindOf $value) "invalid" }}
```

Comment:
If `$value` is empty, what does `kindOf $value` return?

Reply:
I couldn't find any clear documentation, so had to try it out:

| Value as set in `values.yaml` | Result of `kindOf $value` inside that loop
|-----------------------------|------------------------------------------
| `number: 5` | `float64` |
| `string: "value"` | `string` |
| `empty: ""` | `string` |
| `emptyObject: {}` | `map`
| `nullValue: null` | `invalid`